Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM node:lts-alpine AS build
 WORKDIR /app
+ENV PUBLIC_PAYLOAD_URL="/api"
```

Comment:
When this is run in fargate we can pass in the admin dashboard URL. May need to adjust CORS policy or serve both services from the same origin.